apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/FissClaimTransformerV2.java

```diff
@@ -123,15 +131,45 @@ private static Claim transformClaim(RdaFissClaim claimGroup, boolean includeTaxN
    */
   private static List<Claim.SupportingInformationComponent> getSupportingInfo(
       RdaFissClaim claimGroup) {
-    return Strings.isBlank(claimGroup.getFreqCd())
-        ? null
-        : List.of(
-            new Claim.SupportingInformationComponent()
-                .setCategory(createCodeableConcept(C4BBSupportingInfoType.TYPE_OF_BILL))
-                .setCode(
-                    new CodeableConcept(
-                        new Coding(BBCodingSystems.FISS.FREQ_CD, claimGroup.getFreqCd(), null)))
-                .setSequence(1));
+    List<Claim.SupportingInformationComponent> supportingInfo = new ArrayList<>();
+    int sequenceNumber = 1;
+
+    if (Strings.isNotBlank(claimGroup.getFreqCd())) {
+      supportingInfo.add(
+          new Claim.SupportingInformationComponent()
+              .setSequence(sequenceNumber++)
+              .setCategory(createCodeableConcept(C4BBSupportingInfoType.TYPE_OF_BILL))
+              .setCode(
+                  new CodeableConcept(
+                      new Coding(BBCodingSystems.FISS.FREQ_CD, claimGroup.getFreqCd(), null))));
+    }
+
+    if (Strings.isNotBlank(claimGroup.getDrgCd())) {
+      supportingInfo.add(
+          new Claim.SupportingInformationComponent()
+              .setSequence(sequenceNumber)
+              .setCategory(
+                  new CodeableConcept()
+                      .setCoding(
+                          List.of(
+                              new Coding(
+                                  ClaimInformationcategory.INFO.getSystem(),
+                                  ClaimInformationcategory.INFO.toCode(),
+                                  ClaimInformationcategory.INFO.getDisplay()),
+                              new Coding(
+                                  TransformerConstants.CODING_BBAPI_INFORMATION_CATEGORY,
+                                  CCWUtils.calculateVariableReferenceUrl(
+                                      CcwCodebookVariable.CLM_DRG_CD),
+                                  "Claim Diagnosis Related Group Code (or MS-DRG Code)"))))
+              .setCode(
+                  new CodeableConcept(
+                      new Coding(
+                          CCWUtils.calculateVariableReferenceUrl(CcwCodebookVariable.CLM_DRG_CD),
+                          claimGroup.getDrgCd(),
+                          null))));
+    }
+
+    return supportingInfo;
```

Comment:
Previously this method returned `null` rather than an empty list if there was no `freqCd`.  Now it will return an empty list if there is neither a `freqCd` nor a `drgCd`.  Is that intentional?

Reply:
Based on local tests, setting the `supportingInfo` claim attribute to an empty list causes it to not print anything, same as setting it to null